src/main/resources/zenodo-1.0.0-swagger-2.0.yaml

```diff
@@ -524,13 +524,73 @@ paths:
         - preview
       produces:
         - application/json
+      parameters:
+        - in: query
+          name: q
```

Comment:
`q` really? 
Assume this is correct since they had to define it

Reply:
Yep, that's what the name: https://inveniordm.docs.cern.ch/reference/rest_api_drafts_records/#search-records